File: back-end/src/main/java/com/cvaldiviape/dto/response/EntityResponseDTO.java
package com.cvaldiviape.dto.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class EntityResponseDTO {

	private Integer id;
	private String nroDocument;
	private String companyName;
	private String commercialName;
	private String address;
	private String phone;
	private TypeContributorResponseDTO typeContributor;
	private TypeDocumentResponseDTO typeDocument;
	
}

File: front-end/src/app/models/response/entity-response-dto.ts
import { TypeContributorResponseDTO } from "./type-contributor-response-dto";
import { TypeDocumentResponseDTO } from "./type-document-response-dto";

export interface EntityResponseDTO {
  id: number;
  nroDocument: string,
  companyName: string,
  commercialName: string,
  address: string,
  phone: string,
  typeContributor: TypeContributorResponseDTO,
  typeDocument: TypeDocumentResponseDTO,
}